File: Ejercicio1Java/src/mipaquete/Empleado.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package mipaquete;

/**
 *
 * @author Victoria
 */
// Clase Padre Empleado
public class Empleado {

    public String nombre;
    public String apellido;
    private String cedula;
    private double comision_fija;

    // Constructor de Clase Empleado
    public Empleado() {
        this.nombre = "";
        this.apellido = "";
        this.cedula = "";
        this.comision_fija = 0;
    }

    // Metodos set
    public void agregar_nombre(String nombre) {
        this.nombre = nombre;
    }

    public void agregar_apellido(String apellido) {
        this.apellido = apellido;
    }

    public void agregar_cedula(String cedula) {
        this.cedula = cedula;
    }

    public void agregar_comision_fija(double comision_fija) {
        this.comision_fija = comision_fija;
    }

    // Metodos get
    public String obtener_nombre() {
        return nombre;
    }

    public String obtener_apellido() {
        return apellido;
    }

    public String obtener_cedula() {
        return cedula;
    }

    public double obtener_comision_fija() {
        return comision_fija;
    }

    // Metodo para presentar datos
    public String presentar_datos() {
        String cadena = String.format("Información de %s %s\n\tCedula %s", this.obtener_nombre(), this.obtener_apellido(), this.obtener_cedula());
        return cadena;
    }
}

File: ejercicio1/mipaquete/Empleado.py
class Empleado(object):

    def _init_(self):
        self.nombre = ''
        self.apellido = ''
        self.cedula = ''
        self.comision_fija = 0
    
    def agregar_nombre(self, nombre):
        self.nombre = nombre

    def agregar_apellido(self, apellido):
        self.apellido = apellido

    def agregar_cedula(self, cedula):
        self.cedula = cedula

    def agregar_comision(self, comision):
        self.comision_fija = comision
    
    def obtener_nombre(self):
        return self.nombre

    def obtener_apellido(self):
        return self.apellido

    def obtener_cedula(self):
        return self.cedula

    def obtener_comision_fija(self):
        return self.comision_fija
    
    def presentar_datos(self):
        cadena = "Información de %s %s\n\tCedula %s"%(self.obtener_nombre(), self.apellido, self.obtener_cedula())
        return cadena